src/main/java/com/three_tech_solutions/slot_app/controllers/responses/MonthlyFeeDetailsResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.three_tech_solutions.slot_app.controllers.responses;
+
+import com.three_tech_solutions.slot_app.data.enums.MonthlyFeeStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+public class MonthlyFeeDetailsResponse {
+    private UUID id;
+    private int number;
+    private LocalDateTime paymentDate;
+    private double amount;
+    private MonthlyFeeStatus status;
+    private LocalDateTime expirationDate;
```

Comment:
Lo mismo con estas dos fechas


```suggestion
    @JsonFormat(pattern = "dd/MM/yyyy")
    private LocalDateTime paymentDate;
    private double amount;
    private MonthlyFeeStatus status;
    @JsonFormat(pattern = "dd/MM/yyyy")    
    private LocalDateTime expirationDate;
```